Tests/Svg.Benchmark/Svg.Benchmark.csproj

```diff
@@ -1,45 +1,52 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0;netcoreapp3.1;net462</TargetFrameworks>
-    <IsPackable>false</IsPackable>
-    <LangVersion>8.0</LangVersion>
-	<SignAssembly>True</SignAssembly>
-	<AssemblyOriginatorKeyFile>..\Svg.UnitTests\svgkey.snk</AssemblyOriginatorKeyFile>
-  </PropertyGroup>
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFrameworks>net6.0;netcoreapp3.1;net462</TargetFrameworks>
+		<IsPackable>false</IsPackable>
+		<LangVersion>8.0</LangVersion>
+		<SignAssembly>True</SignAssembly>
+		<AssemblyOriginatorKeyFile>..\Svg.UnitTests\svgkey.snk</AssemblyOriginatorKeyFile>
+	</PropertyGroup>

```

Comment:
This one again has move to tabs.

Reply:
Tabs are not intended. 4 spaces is will fix that. It seems on my new developer PC, tap -> spaces is not enabled,